docs/local_setup_sans_docker.md

```diff
@@ -0,0 +1,96 @@
+# Developing Klaxon Locally
+
+This guide is an adaptation of the original [Klaxon developing file](https://github.com/themarshallproject/klaxon/blob/develop/DEVELOPING.md) for use by the Washington Post News Engineering team.
+
+We're assuming that git, homebrew, and [postgres](https://postgresapp.com/) are already installed on your machine.
+
+Start by cloning our fork of the Klaxon repo. For HTTPS:
+
+```
+git clone https://github.com/WPMedia/klaxon.git
+```
+
+For SSH:
+
+```
+git clone git@github.com:WPMedia/klaxon.git
+```
+
+Then, see if you have xcode installed (`xcode-select -version`), and if not, run:
+
+```
+xcode-select --install
+```
+
+If you already have `rbenv` and `ruby-build` install, let's make sure things are up-to-date:
+
+```
+brew update && brew upgrade ruby-build
+```
+
+Otherwise, we'll install `rbenv` and `ruby-build` to manage our ruby environment:
+
+```
+brew install rbenv ruby-build
+```
+
+These next commands will put the proper (albeit aged) version of Ruby that Klaxon requires on your machine and make it available in this repo's directory.
+
+```
+rbenv install
+```
+
+Next, we'll want to make sure our ruby environment is properly initialized by editing our zsh or bash script. Many ways to do that, but if you're new to it, try `open ~/.zshrc` (or `open ~/.bash`) and add the following to the end of the file:
+
+```
+# rbenv addition for Klaxon setup
+eval "$(rbenv init -)"
+```
+
+and then source your updated zsh (or bash) file:
+
+```
+source ~/.zshrc
+```
+
+Next, you'll need to have the proper versions of all of Klaxon's dependency libraries. Run these commands here.
+
+```
+gem install bundler
+bundle install
+```
+
+In order to be allowed to login to Klaxon once it's running on your machine, it'll need to know that your email address is. Create a file named `.env` and paste in these items:
+
+```
+ADMIN_EMAILS="my_awesome_email@gmail.com"
+HOST='localhost:3000'
```

Comment:
🤓 I was curious if the HOST value was getting used at all, if we're setting the PORT. It seems to be! in [config/environments/development.rb](https://github.com/WPMedia/klaxon/blob/develop/config/environments/development.rb#L62)

Reply:
I believe we are! The `HOST` value ended up being important in the local Docker setup PR